personal-growth/learning-english.md

```diff
@@ -0,0 +1,86 @@
+# Learning English
+
+Because all of our communication with clients happens in English, it is extremely important that we
+can all read, write and speak the language fluently. Being able to communicate in a clear and
+concise way is a sign of professionalism and has a tangible impact on the quality of our work.
+
+If you feel like you're still not entirely comfortable with the language, don't worry! We've all
+been there and everyone else at the company is also constantly improving their English. There are
+several ways to improve your skills - here are some of them!
+
+## Professional coaches
+
+There is no replacement for good, old fashioned coaching. A professional coach will not only follow
+your studies and always be there to answer any questions, but they will also be able to create a
+custom learning path for your learning style and the specific contexts where you expect to speak the
+language. While they may seem intimidating at first, coaches are there to help you and are one of
+the best ways to expedite your learning.
+
+A lot of folks here at Nebulab have used and strongly recommend [italki](https://www.italki.com/),
+where you can choose your coach and interact with them remotely, but feel free to choose the online
+or offline platform that best suits you. [It's on us!](https://github.com/nebulab/playbook/blob/master/personal-growth/benefits.md)
+
+We recommend holding at least two 1-hour sessions per week with your coach, however this varies with
+your learning style and experience, so ask your mentor and they'll be able to provide tailored
+guidance.
```

Comment:
I'd specify in this paragraph that it's ok to have lessons during working time, even if maybe we should also allow people to take lessons after work, like in the evening or on Saturday if it's more comfortable for them.

Reply:
@kennyadsl done!